addons/sys_gui/XEH_missionDisplayLoad.sqf

```diff
@@ -3,4 +3,4 @@
 params ["_display"];
 
 // Init the volume control.
-GVAR(MWheel) = _display displayAddEventHandler ["MouseZChanged", QUOTE(_this call FUNC(onVolumeControlAdjust) )];
\ No newline at end of file
+GVAR(MWheel) = _display displayAddEventHandler ["MouseZChanged", {call FUNC(onVolumeControlAdjust)}];
```

Comment:
Yes it does. You can check ACE3 and CBA code in above linka. This actually makes it a tiny bit faster even.